widget/playground/src/components/ExportConfigModal/ExportConfigModal.tsx

```diff
@@ -139,30 +141,18 @@ export function ExportConfigModal(props: ExportConfigModalProps) {
         </HelpLinksContainer>
       </Head>
       <Divider size={30} />
-
       <ButtonsContainer>
-        {Object.keys(typesOfCodeBlocks).map((type, index) => {
-          const key = `block-${index}`;
-          return (
-            <Fragment key={key}>
-              <StyledButton
-                size="medium"
-                disableRipple={true}
-                type={selected === type ? 'secondary' : undefined}
-                variant={selected !== type ? 'contained' : undefined}
-                onClick={() => {
-                  setSelectedIndex(index);
-                  setSelected(type as ExportType);
-                }}>
-                {type}
-              </StyledButton>
-            </Fragment>
-          );
-        })}
-        <BackdropTab
-          css={{
-            transform: `translateX(${TAB_WIDTH * selectedIndex}px)`,
+        <Tabs
```

Comment:
Hover on tabs is not working properly.

<img width="647" alt="image" src="https://github.com/rango-exchange/rango-client/assets/154742563/ca8372e6-0609-4016-9f49-9566b576d625">

<img width="732" alt="image" src="https://github.com/rango-exchange/rango-client/assets/154742563/7316662b-affa-4f7d-ad43-45ee8afe4c82">

Also it has problems in small screens. I think the width of items should not be constant.
<img width="827" alt="image" src="https://github.com/rango-exchange/rango-client/assets/154742563/310b10ee-870d-4f0c-93af-16e51a0a8afd">

Reply:
fixed